### ISSUE TITLE
Replace `print` statements with logger

### ISSUE THREAD

**Issue by wtbarnes** [USNavalResearchLaboratory/eispac#26]
There are several places throughout the package that use `print` statements to output useful information to the screen. Sometimes these are controlled by "debug" or "quiet" arguments. Rather than using these repeated arguments and print statements, we should aim to use a logger (e.g. [the builtin `logging` package]()) so that the amount of output and the level of severity of these messages (DEBUG, INFO, WARN, ERROR) can easily be controlled by the user.

**Comment by MJWeberg** [comment]
Yeah, I have been meaning to refactor everything to use a proper logger. I last setup one up ~6 years ago for a Python 2.7 code, so I will have to brush up on what what has changed since then.

**Comment by wtbarnes** [comment]
I believe the `logging` module is actually largely unchanged. 

However, I'd recommend we use the logger provided by astropy which has a bit of extra functionality. See, for example, how sunpy does this

https://github.com/sunpy/sunpy/blob/62e3f165862879dca7f34a7399f7c1427ed29841/sunpy/__init__.py#L57

https://github.com/sunpy/sunpy/blob/main/sunpy/util/logger.py
